test/fixtures/userBadges/userBadges.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * User's badges response
+ * Multiple responses can be added to the object if required
+ *
+ * @return {Object}
+ */
+
+const userFound = {
+  userExists: true,
+  userBadges: [{ title: 'badgeTitle', description: 'badgeDes' }]
```

Comment:
We can use `badgeDescription` as whole. `badgeDes` might be confusing: design? descending?

Reply:
Got it, will change it